DoxygenHelp.md

```diff
@@ -18,25 +18,31 @@ Alternatively install doxygen and dot yourself:
 
 ### Top-level CMakelists.txt
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in the CMake GUI, CMake will automatically find the doxygen and dot executables. The up-to-date version of the file can be found here:
+
 https://github.com/tumcms/Open-Infra-Platform/blob/development/CMakeLists.txt
 
+In the CMake GUI, the following options can be selected:
+- DOXYGEN_OPTIONAL_COMMENTED_ONLY: If this is not selected, doxygen will assume that all entities are documented, even if there is no documentation available. 

```

Comment:
Rename to DOXYGEN_OPTIONAL_INCLUDE_COMMENTED_ONLY.
Also the description should describe both result (If selected, ... . Otherwise, ...).

Reply:
resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/0c5e14654eea9eceda06c3acf9b832f95a37345d